http-server-netty/src/test/groovy/io/micronaut/http/server/netty/binding/FormDataDiskSpec.groovy

```diff
@@ -0,0 +1,53 @@
+package io.micronaut.http.server.netty.binding
+
+import io.micronaut.context.ApplicationContext
+import io.micronaut.http.HttpRequest
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.HttpStatus
+import io.micronaut.http.MediaType
+import io.micronaut.http.annotation.Body
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Post
+import io.micronaut.http.client.HttpClient
+import io.micronaut.runtime.server.EmbeddedServer
+import reactor.core.publisher.Flux
+import spock.lang.Issue
+import spock.lang.PendingFeature
+import spock.lang.Specification
+
+class FormDataDiskSpec extends Specification {
+    @Issue('https://github.com/micronaut-projects/micronaut-core/issues/6705')
+    @PendingFeature
```

Comment:
Why is the test annotated with `@PendingFeature`?

Reply:
oops, forgot to remove that when i tested for Attr file leaks pre-patch.